sway/config.c

```diff
@@ -921,3 +945,55 @@ void config_update_font_height(bool recalculate) {
 		arrange_root();
 	}
 }
+
+static void translate_bindings(list_t *bindings, list_t *bindsyms,
```

Comment:
This is a little confusing, the only difference between `translate_bindings` and `translate_binding` is the final "s". Can we rename this one to `translate_bindings_list`?

Reply:
Renamed to `translate_binding_list`